cli/cmdlineparser.cpp

```diff
@@ -576,6 +576,15 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
                     mSettings->plistOutput = "./";
                 else if (!endsWith(mSettings->plistOutput,'/'))
                     mSettings->plistOutput += '/';
+
+                if ((mSettings->plistOutput != "./") && (!Path::folderExist(mSettings->plistOutput)))
```

Comment:
The `(mSettings->plistOutput != "./") && ` seems redundant to me.

Reply:
Removed and fixed native path so this can also work in windows.